### ISSUE TITLE
Installs Empty Directory when using CocoaPods

### ISSUE THREAD

**Issue by AndrewBarba** [segment-integrations/analytics-ios-integration-nielsen-dcr#2]
When installing via spec repo, or direct from git, an empty directory is installed and you cannot import the framework. My guess is this has to do with pod spec not defining an `s.source_files`

**Comment by AndrewBarba** [comment]
I made a PR [here](https://github.com/segment-integrations/analytics-ios-integration-nielsen-dcr/pull/3) which fixes the empty directory but when trying to build I get "NielsenAppApi/NielsenAppApi.h file not found". I've added the Nielsen sdk to my build target and can properly import it in my swift code but for some reason the pod isn't finding it. Any idea if there's a build setting I need to update?
